src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -64,6 +64,12 @@ public class ParameterProvider {
   public static final Constants.BdecParquetCompression BDEC_PARQUET_COMPRESSION_ALGORITHM_DEFAULT =
       Constants.BdecParquetCompression.GZIP;
 
+  // Iceberg mode default parameters
+  public static final int MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST_ICEBERG_MODE_DEFAULT = 1;
+
+  // If the provided parameters need to be verified and modified to meet Iceberg mode
+  private final boolean isIcebergMode;
```

Comment:
what happens if customer create a Iceberg client but override this value to false?

Reply:
I don't think client can inject `ParameterProvider` directly, instantiate is called by `ClientInternal` at [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/6c9bc9b3e3b9bd687898aeb59cb0d30940b4c8c9/src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java#L172).